### ISSUE TITLE
automl / learning rate should be parametric

### ISSUE THREAD

**Issue by gionatamettifogo** [analitico/analitico-automl#17]
instead of passing an optimizer by string, eg "Adam" we should build an optimizer object with a parametric learning rate

**Comment by gionatamettifogo** [comment]
eg. 
https://www.tensorflow.org/api_docs/python/tf/estimator/DNNLinearCombinedRegressor?version=stable#example_2

# To apply learning rate decay, you can set dnn_optimizer to a callable:
lambda: tf.keras.optimizers.Adam(
    learning_rate=tf.compat.v1.train.exponential_decay(
        learning_rate=0.1,
        global_step=tf.compat.v1.train.get_global_step(),
        decay_steps=10000,
        decay_rate=0.96)
